Tests/OpenWeatherMap.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Resource  ../Resources/PO/DesktopShared.robot
+Resource  ../Resources/PO/LoginPage.robot
+Resource  ../Resources/PO/TopNav.robot
+Resource  ../Resources/CommonWeb.robot
+Test Setup  Begin Web Test
+Test Teardown  End Web Test
+# robot  -d results tests/openweathermap.robot
+
+
+*** Variables ***
+${BROWSER} =  ff
+${URL} =  https://openweathermap.org/
+${VALID_ USER_EMAIL} =  hanna.maniak19@gmail.com
+${VALID_USER_PASSWORD} =  52Mon1489!
+
+
+*** Test Cases ***
+Succussfull Login
+    Sign In With Valid Credentials
```

Comment:
That is pretty bad sample of test, we need to do actions and check without hiding it inside keywords